azext_iot/central/commands_device.py

```diff
@@ -69,13 +69,29 @@ def registration_info(
     token=None,
     central_dns_suffix="azureiotcentral.com",
     device_status=None,
+    max_devices=None,
+    summarize_registration=False,
 ):
     provider = CentralDeviceProvider(cmd=cmd, app_id=app_id, token=token,)
     if not device_id:
-        return provider.get_all_registration_info(
+        if summarize_registration:
```

Comment:
the fact that this code path is so specific leads me to think it deserves its own command